src/huggingface_hub/_snapshot_download.py

```diff
@@ -145,6 +147,9 @@ def snapshot_download(
 
     </Tip>
     """
+    if HF_HUB_ETAG_TIMEOUT != DEFAULT_ETAG_TIMEOUT:
+        etag_timeout = HF_HUB_ETAG_TIMEOUT
```

Comment:
```suggestion
```

I would rather prefer to have this logic only in `http_get` and `_request_wrapper`

Reply:
@Wauplin I am a bit confused (: 
First, It seems like http_get uses the as well _request_wrapper.
So according to your suggestion above and at the bottom.
The timeout in http_get should be set according HF_HUB_DOWNLOAD_TIMEOUT
The timeout in _request_wrapper should be set according HF_HUB_ETAG_TIMEOUT
So if both will be set the HF_HUB_ETAG_TIMEOUT in _request_wrapper will override the HF_HUB_DOWNLOAD_TIMEOUT.
Maybe we should use only one variable in _request_wrapper, lets say HF_HUB_REQUEST_TIMEOUT and thats all?